src/actions.js

```diff
@@ -381,6 +381,7 @@ export const updateSystemUseNotice = (displayUseWarning) => (dispatch) => {
   });
 };
 
+// eslint-disable-next-line max-len
```

Comment:
Instead of adding a comment to disable the linter here, is there a reason for not adding line breaks to the function? Would something like this work? 

```
export const displaySystemUseNotice = (authenticated) => (dispatch, getState) =>
  dispatch(
    checkIfDisplaySystemUseNotice(
      authenticated,
      getState().popups.systemUseWarnPopup
    )
  );


```